openmetadata-service/src/test/java/org/openmetadata/service/alerts/AlertsRuleEvaluatorTest.java

```diff
@@ -89,16 +92,17 @@ void test_matchAnyEntityFqn(TestInfo test) throws IOException {
 
   @Test
   void test_matchAnyEntityId(TestInfo test) throws IOException {
-    // Create Table
-    List<Column> columns = List.of(new Column().withName("c1").withDataType(ColumnDataType.INT));
+    // Create Table Entity
+    List<Column> columns = List.of(TableResourceTest.getColumn("c1", ColumnDataType.INT, null));
     CreateTable create = tableResourceTest.createRequest(test).withColumns(columns);
```

Comment:
Feel free to static import the method to make the getColumn() method call invokes less verbose...